canopy/canopyjs/src/transactions/addressing.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const crypto = require('crypto');
+
+// The namespace registry prefix for global state (00ec00)
+const NAMESPACE_REGISTRY_PREFIX = '00ec00';
+
+// The contract prefix for global state (00ec02)
+const CONTRACT_PREFIX = '00ec02';
```

Comment:
These magic strings should probably come from some kind of config file instead of constants living with business logic, no?

Reply:
It's a very sabre-specific constant, so it doesn't make sense to be in a config file. Any changes to these values would probably also require code changes here.